<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Train_Dialog</class>
 <widget class="QWidget" name="Train_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>822</width>
    <height>327</height>
=======
    <width>818</width>
    <height>794</height>
>>>>>>> a9988750f95b7e7d529ffe687b927c26d586a526
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="5" column="2">
      <widget class="QPushButton" name="Run_pushButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_train">
       <property name="text">
        <string>검증 비율</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_test">
       <property name="text">
        <string>테스트 비율</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="Test_pushButton">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="verification_rate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>학습 기법</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="test_rate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="studying_rate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_study">
       <property name="text">
        <string>학습 비율</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="model_save_pushButton">
       <property name="text">
        <string>모델 저장</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QTabWidget" name="study_tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>ARIMA</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <layout class="QGridLayout" name="ARIMA_gridLayout">
           <item row="1" column="2">
            <widget class="QLabel" name="label_d">
             <property name="text">
              <string>d (차분)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_q">
             <property name="text">
              <string>q (이동평균)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_p">
             <property name="text">
              <string>p (자기상관)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="6">
            <widget class="QComboBox" name="c_nc_comboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>c</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>nc</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="4">
            <widget class="QSpinBox" name="q_spinBox">
             <property name="maximum">
              <number>9</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="p_spinBox">
             <property name="maximum">
              <number>9</number>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ㅁㄴㅇㄹ&lt;/p&gt;&lt;p&gt;ㅁㄴㅇㄹ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;c : 상수 포함&lt;/p&gt;&lt;p&gt;nc : 상수 미포함&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="d_spinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>2</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_x">
             <property name="text">
              <string>x 축</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_x"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_y">
             <property name="text">
              <string>y 축</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QComboBox" name="comboBox_y"/>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="adf_test_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="auto_arima_pushButton">
           <property name="text">
            <string>자동설정</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="rmse_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="arima_groupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGraphicsView" name="arima_graphicsView"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>LSTM</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="2" column="7">
          <widget class="QComboBox" name="AdjClose_combobox"/>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0" colspan="11">
          <layout class="QGridLayout" name="gridLayout_9"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="Low_combobox"/>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Adj Close</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>High</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Learning Rate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="9">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>모델 이름</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Sequence Length</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="9">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Epochs</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="Epochs_linetext"/>
         </item>
         <item row="0" column="7">
          <widget class="QComboBox" name="High_combobox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item row="2" column="10">
          <widget class="QComboBox" name="Volume_combobox"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLineEdit" name="LearningRate_linetext"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QLineEdit" name="SequenceLength_linetext"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="Date_combobox"/>
         </item>
         <item row="2" column="4">
          <widget class="QComboBox" name="Close_combobox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Low</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="Open_combobox"/>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="10">
          <widget class="QPushButton" name="Train_pushButton">
           <property name="text">
            <string>학습 실행</string>
           </property>
          </widget>
         </item>
         <item row="4" column="10">
          <widget class="QLineEdit" name="ModelName_lineText"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>studying_rate</tabstop>
  <tabstop>verification_rate</tabstop>
  <tabstop>test_rate</tabstop>
  <tabstop>comboBox_x</tabstop>
  <tabstop>comboBox_y</tabstop>
  <tabstop>p_spinBox</tabstop>
  <tabstop>d_spinBox</tabstop>
  <tabstop>q_spinBox</tabstop>
  <tabstop>c_nc_comboBox</tabstop>
  <tabstop>auto_arima_pushButton</tabstop>
  <tabstop>model_save_pushButton</tabstop>
  <tabstop>Run_pushButton</tabstop>
  <tabstop>Test_pushButton</tabstop>
  <tabstop>Date_combobox</tabstop>
  <tabstop>Open_combobox</tabstop>
  <tabstop>High_combobox</tabstop>
  <tabstop>Low_combobox</tabstop>
  <tabstop>Close_combobox</tabstop>
  <tabstop>AdjClose_combobox</tabstop>
  <tabstop>Volume_combobox</tabstop>
  <tabstop>Epochs_linetext</tabstop>
  <tabstop>LearningRate_linetext</tabstop>
  <tabstop>SequenceLength_linetext</tabstop>
  <tabstop>ModelName_lineText</tabstop>
  <tabstop>Train_pushButton</tabstop>
  <tabstop>arima_graphicsView</tabstop>
  <tabstop>study_tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
